content.en/tools/nebula/_index.md

```diff
@@ -0,0 +1,109 @@
+![Nebula Logo](./nebula.png)
+
+# Nebula
+
+Nebula is a [libp2p](https://github.com/libp2p/go-libp2p-kad-dht) DHT crawler
+and monitor that tracks the liveliness of peers. The crawler connects
+to [DHT](https://en.wikipedia.org/wiki/Distributed_hash_table) bootstrap peers
+and then recursively follows all entries in
+their [k-buckets](https://en.wikipedia.org/wiki/Kademlia) until all peers have
+been visited. The crawler supports
+the [IPFS](https://ipfs.network), [Filecoin](https://filecoin.io), [Polkadot](https://polkadot.network/), [Kusama](https://kusama.network/), [Rococo](https://substrate.io/developers/rococo-network/), [Westend](https://wiki.polkadot.network/docs/maintain-networks#westend-test-network)
+networks [and more](https://github.com/dennis-tra/nebula/blob/a33a5fd493caaeb07e92ecc73c32ee87ae9e374f/pkg/config/config.go#L11).
```

Comment:
(nit) maybe make a bullet list?

Reply:
This looks much better in non-markdown 👍